ci/lint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+SCRIPT_PATH=${0%/*}
+if [ "$SCRIPT_PATH" = "$0" ]; then
+	cd ..
+fi
+./ci/commoncmd.sh
+yarn install --frozen-lockfile
```

Comment:
```suggestion
./ci/commoncmd.sh
yarn install --frozen-lockfile
```

I don't think you need either of these lines (certainly not the `yarn install` one as that's done in config.yml)